examples/acquia-8.0/.lando.yml

```diff
@@ -0,0 +1,9 @@
+name: acquia-80
+recipe: acquia
+config:
+  php: "8.0"
+  acli_version: 1.30.0
```

Comment:
Should this version be enclosed in double quotes since it ends in `.0`?  I know js tends to truncate these numbers which could create an issue.

Reply:
I believe quotes or no quotes should both be fine based on my experiences with the YAML interpretation.